contract/src/lib.rs

```diff
@@ -177,7 +178,7 @@ impl NearCLP {
         let caller = env::predecessor_account_id();
         let shares_minted;
         let near_amount = env::attached_deposit();
-        let token_amount = max_token_amount;
+        let computed_token_amount;
```

Comment:
why `computed_` prefix?

Reply:
because is the token amount computed by the core swap function, to balance the pool.
it's not a token amount indicated by the user, it is computed by the contract